src/test/kotlin/net/maizegenetics/phgv2/cli/FullPipelineIT.kt

```diff
@@ -148,6 +146,18 @@ class FullPipelineIT {
             assertTrue(asmDiff < 0.00001, "${asmName} Fasta is not the same as input")
         }
 
+        //what is in agc?
+        val genomes = retrieveAgcData(TestExtension.testTileDBURI, listOf("listset"))
+
+        if (genomes != null) {
+            println("agc genomes: ${genomes.joinToString(",")}")
+            val cmd = mutableListOf("listctg")
+            cmd.addAll(genomes)
+            println("agc contigs: ")
+            retrieveAgcData(TestExtension.testTileDBURI, cmd)?.forEach { println(it) }
```

Comment:
Does this need an assertion?

Reply:
No assertion needed because I deleted that code, which I had meant to do but forgot.